### ISSUE TITLE
Incorrect calculation for "Enforces HTTPS"

### ISSUE THREAD

**Issue by mpreissner** [dhs-ncats/pshtt#207]
# 🐛 Bug Report

## To Reproduce

Install pshtt on CentOS 7.7.
Run test against desired site with known Valid HTTPS and Defaults to HTTPS

## Expected behavior

A given site returns "Valid HTTPS=True" and "Defaults to HTTPS=True", so "Domain Enforces HTTPS" should be True.

I support a federal agency...according to what's been published, pshtt is supposed to calculate "Domain Enforces HTTPS" based on (Domain Supports HTTPS=True AND (Defaults to HTTPS=True OR (Strictly Forces HTTPS=True AND Redirect=True))).  If this logic is correct, then any domain with Valid HTTPS=True and Defaults to HTTPS=True should return True for Domain Enforces HTTPS, regardless of the values for Strictly Forces HTTPS and Redirect.

Test site was "list.ahrq.gov".

## Any helpful log output

Paste the results here:

```bash

```

**Comment by DOS-cyber** [comment]
Remember that it checks [site].[tld] as well as [www].[site].[tld]. http://www.list.ahrq.gov returns a 404, it doesn’t 3xx redirect to https.
Neil

**Comment by mpreissner** [comment]
Thanks Neil.  If we simply get rid of the www 4th level domain, will that make the calculation come up as desired?

**Comment by echudow** [comment]
You're right, the documentation should be updated.  https://github.com/cisagov/pshtt/pull/192 updated the logic for Domain Enforces HTTPS to also require Strictly Forces HTTPS to be True.
